plugins/modules/registration_command.py

```diff
@@ -65,6 +65,7 @@
   lifecycle_environment:
     description:
     - Lifecycle environment for the host.
+    - Ignored since Katello 4.12.
```

Comment:
Is this because a host can now be in multiple LCEs? Is there a replacement we can point to?

Reply:
The AK drives the LCE(s)

Reply:
Can we include that in the message?

Reply:
Sure